scheduler/example-prod-config.edn

```diff
@@ -19,6 +19,11 @@
                                 :memory-gb 96
                                 :retry-limit 15
                                 :cpus 20}}
+ :executor {:command "./cook-executor"
+            :uri {:cache true
+                  :executable true
+                  :extract false
+                  :value #config/env "COOK_EXECUTOR"}}
```

Comment:
Elsewhere in this file, we use actual (fake) values, instead of environment variables. Maybe we should follow that pattern here